wremnants/include/utils.h

```diff
@@ -5,6 +5,11 @@
 #include <eigen3/Eigen/Dense>
 #include <eigen3/unsupported/Eigen/CXX11/Tensor>
 #include "defines.h"
+#include "ROOT/RVec.hxx"
+#include "TLorentzVector.h"
```

Comment:
TLorentzVector is known to be quite slow. For now it is fine, but once the code is finalised it might be better to move to something else. For example see here:
https://root.cern.ch/doc/master/classTLorentzVector.html

If you only need the projections in the transverse plane you can probably also work just with TVector2, using pt and phi